contrib/librespot.service

```diff
@@ -9,7 +9,7 @@ User=nobody
 Group=audio
 Restart=always
 RestartSec=10
-ExecStart=/usr/bin/librespot -n "%p on %H"
+ExecStart=/bin/sh -c 'exec /usr/bin/librespot --name librespot@$(hostname -s)'
```

Comment:
Sorry, hadn't seen #200 before, thanks for reporting the issue. So I don't really mind anymore, and if/when #200 will be resolved, the workaround can still be made undone.